README.md

```diff
@@ -34,7 +34,17 @@ This lab might seem a bit different than what you've seen before. Take a look at
 
 ### A Note on Testing
 
-In the `sql_queries.rb`, the select statements you write are being queried on the table you create in `create.sql`, but **not** on the data you've inserted via `insert.sql`. Instead it's on our own inserted data that's been encoded behind the scenes. 
+Let's briefly go over what is happening in the `before` block that our tests will be using.
+
+```ruby
+before do
+  @db = SQLite3::Database.new(':memory:')
+  @sql_runner = SQLRunner.new(@db)
+  @sql_runner.execute_create_file
+end
+```
+Before each test, it creates a new in-memory database as well as a new instance of the SQLRunner class. The SQLRunner class lives in your `bin` directory and was created to help connect to the database. The code in your test suite can now run against your in-memory database, keeping it separate from you real database.
```

Comment:
why do we make an in-memory store? It's because we dpon't want one running of the tests to affect another running of the tests. What if our tests add's ten items! Now those are in there forever. An in-memory store means that it gets trashed after every running of the tests